tests/Statable/StatableArticle.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Sebdesign\SM\Test\Statable;
+
+use Sebdesign\SM\Traits\Statable;
+
+class StatableArticle
+{
+    use Statable;
+
+    const HISTORY_MODEL = [
+        'name' => 'Sebdesign\SM\Test\Statable\ArticleState',
+        'foreign_key' => 'article_id',
+    ];
+    const SM_CONFIG = 'graphA'; // the SM graph to use
```

Comment:
This should be a property or a function like `getStateMachineGraph()` to be more extendible.
Maybe the graph is stored in the database, or I want to set a different one for an instance.